formver.compiler-plugin/uniqueness/src/org/jetbrains/kotlin/formver/uniqueness/UniqueCheckVisitor.kt

```diff
@@ -80,43 +81,66 @@ object UniqueCheckVisitor : FirVisitor<Pair<UniqueLevel, UniquePathContext?>, Un
         return Pair(UniqueLevel.Shared, null)
     }
 
+    private fun verifyPassingRules(
```

Comment:
It's not obvious to me why a function called "verify" returns a Trie, could you add a comment?